scss/titles.scss

```diff
@@ -0,0 +1,5 @@
+.title {
+  font-family: "Roboto", sans-serif;
```

Comment:
This is really good but we need to add the Roboto font since it's not a default browser font, for that we need to import it from google fonts and add it to the application.scss